### PR TITLE
Project management update and fork

### LINKED ISSUE #8: [BUG] - unable to publish packages to pypi and testpypi
Issue is that CI/CD pipeline is broken
unable to test new packages on test/ branches or to publish initial beta release of streamlit-analytics2 to pypi

### PR DESCRIPTION
**Pull request by 444B**
# Pull Request Template

## TL;DR
Provide a brief summary of the changes made.
- Have potentially fixed the pypi packaging issues. the problem may have been from incorrect toml formatting
- adopted pyproject.toml as a setup.py is not current industry best practice
- also moved the files that form streamlit-analytics2 into a new folder called src/. this is to more easily keep track of what is getting compiled into the package

## Problem Solved
1. Describe the problem your changes address. 
2. Please mention the issue number if relevant
- potentially allows the proper building and distribution of files via pypi
- This is intended to fix #8

### CODE REVIEW

**Comment by 444B** [comment]
After merge, we will be able to see if the CI/CD pipeline is back up.
SUCCESS: we will see a package called streamlit-analytics2 on pypi [link](https://pypi.org/project/streamlit-analytics2/)

Failure: fail to build, issue with keys or repo setup, no beta package on pypi
